src/poliastro/tests/test_czml.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from numpy.testing import assert_allclose
+
+from poliastro.czml.extract_czml import CZMLExtractor
+from poliastro.examples import iss, molniya
+
+
+def test_czml_add_orbit():
+    start_epoch = iss.epoch
+    end_epoch = iss.epoch + molniya.period
+
+    Extractor = CZMLExtractor(start_epoch, end_epoch, 10)
+
+    Extractor.add_orbit(
+        molniya, label_text="Molniya", label_fill_color=[125, 80, 120, 255]
+    )
+    Extractor.add_orbit(iss, label_text="ISS", path_show=False)
+
+    cords_molniya = Extractor.czml[0]["position"]["cartesian"][4 * 5 : 4 * 6]
```

Comment:
Instead of testing a small subset, I would choose one of two options:

1. Checking that the full coordinates list is equal to the appropriate `propagate(...)`
2. Don't check the values at all, and only that `["position"]["cartesian"]` has the expected number of elements

I don't think checking the actual values is that important in this case. In general, let's keep the test simple to follow and understand.